test/src/tests/generated/lro/lros.go

```diff
@@ -64,10 +64,6 @@ func (client LROsClient) Delete202NoRetry204Sender(req *http.Request) (future LR
 	if err != nil {
 		return
 	}
-	err = autorest.Respond(resp, azure.WithErrorUnlessStatusCode(http.StatusOK, http.StatusAccepted))
```

Comment:
so the purpose of this line is just to check if there was something else than those statuses ?

Reply:
Correct.  The status codes here come from the `responses` section in the swagger, however there are a handful of well-defined status codes for LROs as defined by ARM and we check them in the LRO state machine so having this check is redundant plus it prevents the returned future from containing the underlying HTTP response.